src/types/KongSwap.ts

```diff
@@ -0,0 +1,42 @@
+import {
+    LPTokenReply,
+    PoolsReply,
+    SwapAmountsReply,
+    SwapReply,
+} from "./actors/kongswap/kongBackend";
+import { Token as GeneralToken } from "./ISwap";
+
+export type Token = GeneralToken & {
+    fee: bigint;
+    decimals: number;
+    token: string;
+    token_id: number;
+    chain: string;
+    canister_id: string;
+    icrc1: boolean;
```

Comment:
Consider using enums for such stuff: https://www.typescriptlang.org/docs/handbook/enums.html